### ISSUE TITLE
Release proceedure for Wasserstein `v1.1.1`

### ISSUE THREAD

**Issue by matthewfeickert** [thaler-lab/Wasserstein#7]
> now that this has gotten moved over to the `thaler-lab` GitHub org, could we work together to make a `v1.1.1` release? (c.f. https://github.com/thaler-lab/Wasserstein/pull/5#issuecomment-1312059614).

_Originally posted by @matthewfeickert in https://github.com/thaler-lab/Wasserstein/issues/5#issuecomment-1776499096_

Following PR #5, this Issue can be used by @rikab and @matthewfeickert to try to coordinate the steps necessary to update the source code version information, make a GitHub Release, build all the wheels necessary, and then publish them to PyPI for a Wasserstien `v1.1.1` release.

If @pkomiske has any release notes lying around those would be great. If not, this Issue can get turned into the start of some maintainer release docs.

**Comment by rkansal47** [comment]
Hi @matthewfeickert, have you looked into this further and is there some way I can help, particularly in releasing Python 3.11 wheels?

**Comment by matthewfeickert** [comment]
@rkansal47 No, I haven't yet, though thanks for the reminder that this is a todo.

First things I would recommend doing:

* [ ] Having @pkomiske give you maintainer permissions on PyPI (currently https://pypi.org/project/Wasserstein/ is jsut @pkomiske).
* [ ] Split the CI components from the wheel building in CI
* [ ] Have the wheel building workflows run and [upload the wheels to GitHub Actions as an artifact](https://learn.scientific-python.org/development/guides/gha-wheels/#the-core-job-3-main-oss)
* [ ] Have a seperate publishing workflow that downloads the artifacts and then publishes them using [PyPI's Trusted Publishers](https://learn.scientific-python.org/development/guides/gha-wheels/#publishing).

**Comment by matthewfeickert** [comment]
cc @rikab for the above (I forgot to add you earlier).

**Comment by j-s-ashley** [comment]
> * [ ] Have the wheel building workflows run and [upload the wheels to GitHub Actions as an artifact](https://learn.scientific-python.org/development/guides/gha-wheels/#the-core-job-3-main-oss)

Completed with #26 (and #43) 👍🏻

**Comment by matthewfeickert** [comment]
> * [ ] Have a seperate publishing workflow that downloads the artifacts and then publishes them using [PyPI's Trusted Publishers](https://learn.scientific-python.org/development/guides/gha-wheels/#publishing).

(Double posting across here and email &mdash; sorry in advance :grimacing:)

@rikab Thanks to great work by @j-s-ashley we're imminently close to being ready to publish new patch release wheels for EnergyFlow and Wasserstein to PyPI with fixes and new platform support added. Part of this includes transitioning publishing to use the more secure Trusted Publishers workflow.

This requires that either you link the PyPI projects to the GitHub repositories, or that you add @j-s-ashley to the PyPI projects with owner permissions (at least temporarily) for them to do it. Can you help us with this, ideally this week if possible?

**Comment by 100rab-S** [comment]
Hey @matthewfeickert, thank you for your efforts in bringing this fix forward. Are we any closer to publishing `v1.1.1`?
